core-java-modules/core-java-concurrency-simple/src/test/java/com/baeldung/concurrent/completablefuture/CompletableFutureTimeoutUnitTest.java

```diff
@@ -0,0 +1,83 @@
+package com.baeldung.concurrent.completablefuture;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.concurrent.*;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+public class CompletableFutureTimeoutUnitTest {
+    private static WireMockServer wireMockServer;
+    private static ScheduledExecutorService executorService;
+    private static final int DEFAULT_TIMEOUT = 1000; //1 seconds
+    private static final int TIMEOUT_STATUS_CODE = 408; //0.5 seconds
+
+    @BeforeAll
+    public static void setUp() {
```

Comment:
Note: as this is JUnit 5, we don't need the class or methods to have `public` on them.